packages/turf-kinks/package.json

```diff
@@ -66,7 +66,6 @@
   "dependencies": {
     "@turf/helpers": "workspace:^",
     "@types/geojson": "^7946.0.10",
-    "sweepline-intersections": "^1.5.0",
```

Comment:
Does this mean we should also revert the `sweepline-intersections` change in `@turf/line-intersect` too?